packages/functional-tests/tests/misc/cookiesDisabled.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect } from '../../lib/fixtures/standard';
+
+// there is no way to disable cookies using wd. Add `disable_cookies`
```

Comment:
Silly question but what is wd? Also should this be a note about local storage and the `disable_local_storage` param instead? I don't see anything `disable_cookies` in our codebase.

Whenever we do actually have this working with cookies being disabled (noted in FXA-6704 that prod just breaks with a 500 error if they're disabled), maybe it's possible to write a unit test for this since we do have [some already](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/pages/CookiesDisabled/index.test.tsx#L144) when rendering just that page. Based on the [Playwright docs](https://playwright.dev/docs/api/class-browsercontext#browser-context-set-extra-http-headers), I'm kind of wondering if it's possible to create a new browser context and set `extraHTTPHeaders` to `{ 'Cookie': '' }` but I have no idea if that'd work (and it would definitely fail right now, since content-server shows a 500 error with them disabled).

Reply:
The comment line had been just copied over from the old tests but yeah I can make the change in comment for `disable_local_storage`.
Yeah a unit test sounds good for now atleast.